src/common/constants.js

```diff
@@ -60,3 +61,16 @@ export const p2p_playstore_url = 'https://play.google.com/store/apps/details?id=
 export const pushwoosh_app_code = 'DD293-35A19'
 export const sample_rate = 25
 export const zoho_url = 'https://deriv.zohorecruit.com'
+export const localized_link_url = {
```

Comment:
Can we use Object.freeze, so that this can't be modified from other files

Reply:
will do~